csource/common.go

```diff
@@ -106,6 +114,79 @@ static void install_segv_handler()
 	sigaction(SIGBUS, &sa, NULL);
 }
 
+#ifdef __NR_syz_emit_ethernet
+#define COMMAND_MAX_LEN 64
+
+static void execute_command(const char *format, ...)
+{
+	va_list args;
+	int rv;
+	char command[COMMAND_MAX_LEN];
+
+	va_start(args, format);
+
+	rv = vsnprintf(&command[0], sizeof(command), format, args);
+	if (rv < 0) {
+		perror("snprintf()");
+		exitf("tun: snprintf failed");
+	}
+	if ((unsigned)rv >= sizeof(command)) {
+		fail("tun: command \"%s\"doesn't fit into buffer", &command[0]);
+	}
+	if (system(&command[0]) < 0) {
+		perror(&command[0]);
+		exitf("tun: command \"%s\" failed", &command[0]);
+	}
+
+	va_end(args);
+}
+
+int tunfd;
+
+#define LOCAL_MAC "42:42:42:42:42:42"
```

Comment:
Will it be possible to support several parallel executors so that they don't collide?